### PR TITLE
Create Repository and Dataset without owner, workspace, workspaceType

### PR DESCRIPTION
**Pull request by anandjakhaniya**
Splitted PR: https://github.com/VertaAI/modeldb/pull/1804

### CODE REVIEW

**Comment by anandjakhaniya** [comment]
Splitted as much as. Now this PR only content the create flow repository and dataset.

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 1]
```diff
@@ -679,11 +678,11 @@ public static void logAmazonServiceExceptionErrorCodes(Logger LOGGER, AmazonServ
   }
 
   public static ResourceVisibility getResourceVisibility(
-      Optional<WorkspaceType> workspaceType, ProtocolMessageEnum visibility) {
-    if (!workspaceType.isPresent()) {
+      Workspace workspace, ProtocolMessageEnum visibility) {
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: I'd recommend leaving Optional in. It's _much_ safer than something that might be null. It's also much more obvious in terms of api that something is optional vs required.

**Review by conradoverta — APPROVED** [review]
LGTM

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java]: Change it to optional now.

**Pull request merged**